src/intTest/resources/success_uat_data/NHS002/full_acceptance_without_nhsack.edifact.dat

```diff
@@ -0,0 +1,28 @@
+UNA:+.? '
+UNB+UNOB:2+110000044300001:80+110002063600005:80+030304:2347+428090++MEDRPT'
+UNH+000001+MEDRPT:0:1:RT:NHS002'
+BGM+LSR'
+DTM+137:200303042347:203'
+S01+01'
+NAD+SLA+++TRELISKE HAE/CH'
+SPR+DPT'
+S01+01'
+NAD+SLA+REF00:902'
```

Comment:
This is actually `NAD+SLA+REF00:903'` in the edifact test files, but `HealthcareRegistrationIdentificationCode` `903` does not currently exist